qiskit_aqt_provider/aqt_resource.py

```diff
@@ -309,6 +321,15 @@ def __init__(
         resource_name: str,
         noisy: bool,
     ) -> None:
+        """Initialize an offline simulator resource.
+
+        Args:
+            provider: Qiskit provider that owns this backend.
+            workspace_id: name of the AQT workspace the mapped resource belongs to.
+            resource_id: name of the resource in the AQT cloud portal.
+            resource_name: pretty name of the resource in the AQT cloud portal.
+            noisy: whether to configure a noise model in the simulator backend.
```

Comment:
we called the "simulator with noise" such that we avoid "noisy simulator" phrases, because this sounds a bit sloppy and awkward. maybe replace `noisy` argument with `use_noise_model`?

Reply:
Sure. But I'd like to defer this to a separate PR, since this would be a breaking API change.